apps/portals/src/configurations/elportal/synapseConfigs/projects.ts

```diff
@@ -25,14 +25,14 @@ export const projectCardConfiguration: CardConfiguration = {
     title: 'name',
     subTitle: 'principalInvestigators',
     description: 'abstract',
-    secondaryLabels: ['institutions', 'program', 'grantNumber'],
+    secondaryLabels: ['institutions', 'program', 'grant'],
   },
   secondaryLabelLimit: 4,
   titleLinkConfig: {
     isMarkdown: false,
     baseURL: 'Explore/Projects/DetailsPage',
-    URLColumnName: 'grantNumber',
-    matchColumnName: 'grantNumber',
+    URLColumnName: 'grant',
+    matchColumnName: 'grant',
```

Comment:
Card title on explore page should link to the details page using the `grant` value(s) as the unique identifier for the project.